force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.js

```diff
@@ -52,6 +36,8 @@ export default class ServiceScheduleCreator extends NavigationMixin(LightningEle
             this.originalModel = result;
```

Comment:
One question here. Why do we set the originalModel to result and not result.data.

Reply:
Originally, I was thinking about the possibility of needing to use refreshApex but since we don't I can store data instead. Great eye :)